app/src/main/java/com/xmartlabs/scasas/criminalintent/ui/common/OnTappedListener.java

```diff
@@ -0,0 +1,9 @@
+package com.xmartlabs.scasas.criminalintent.ui.common;
+
+/**
+ * Created by scasas on 2/13/17.
+ */
+public interface OnTappedListener<T> {
```

Comment:
Action is a rx interface, but rx is not integrated yet. maybe you can rename the class